src/shadowbox/integration_test/test.sh

```diff
@@ -117,9 +117,10 @@ function cleanup() {
   (docker exec "${CLIENT_CONTAINER}" curl --silent --connect-timeout 5 "${TARGET_IP}" > /dev/null && \
     fail "Client should not have access to target IP") || (($? == 28))
 
-  # Exit code 6 for "Could not resolve host".
-  (docker exec "${CLIENT_CONTAINER}" curl --silent --connect-timeout 5 http://target > /dev/null && \
-    fail "Client should not have access to target host") || (($? == 6))
+  # Exit code 6 for "Could not resolve host".  In some environments, curl reports a timeout
+  # error (28) instead, which is surprising.  TODO: Investigate and fix.
```

Comment:
Could you add the error string in the comment too?

Reply:
It's not interesting.  You just see that the equality test failed.  Or am I misinterpreting what you're asking for?

Reply:
Nevermind. You already said it's a timeout.